server/routes/register.js

```diff
@@ -29,6 +31,13 @@ router.post("/", async function(req, res) {
       );
       res.cookie("token", token, { httpOnly: true });
       res.status(201).send();
+
+      const userProfile = await new Profile({ user: userDoc.id, firstName: "", lastName: "", phone: "", location: "", imageUrl: ""}).save()
```

Comment:
- Unused variable userProfile
- This line could be shortened if on the Profile model you set `default: ""` for firstName, lastName, etc. and then you wouldn't have to include those default values here.